components/builder-web/npm-shrinkwrap.json

```diff
@@ -4,20 +4,20 @@
   "dependencies": {
     "@angular/animations": {
       "version": "4.3.6",
-      "from": "@angular/animations@>=4.3.4 <5.0.0",
+      "from": "https://registry.npmjs.org/@angular/animations/-/animations-4.3.6.tgz",
```

Comment:
Was this shrinkwrapped with npm 2.15.1?

Reply:
(I just ask because I wouldn't expect some of these to have changed in quite this way, but if so, :ignoreme:.)

Reply:
I probably had npm 5 running when I shrinkwrapped it. I'll fix up the lockfile with the correct version